suricata/update/loghandler.py

```diff
@@ -62,18 +64,58 @@ def emit(self, record):
             level_prefix = self.YELLOW
             message_prefix = ""
 
-        self.stream.write("%s%s%s - <%s%s%s> -- %s%s%s\n" % (
-            self.GREEN,
-            self.formatTime(record),
-            self.RESET,
-            level_prefix,
-            record.levelname.title(),
-            self.RESET,
-            message_prefix,
-            self.mask_secrets(record.getMessage()),
-            self.RESET))
+        if os.isatty(sys.stdout.fileno()) and \
+                record.levelname == "INFO" or record.levelname == "DEBUG":
+            self.stream.write("%s%s%s - <%s%s%s> -- %s%s%s\n" % (
+                self.GREEN,
+                self.formatTime(record),
+                self.RESET,
+                level_prefix,
+                record.levelname.title(),
+                self.RESET,
+                message_prefix,
+                self.mask_secrets(record.getMessage()),
+                self.RESET))
+        elif os.isatty(sys.stderr.fileno()) and \
+                record.levelname == "WARNING" or record.levelname == "ERROR":
+            self.stream.write("%s%s%s - <%s%s%s> -- %s%s%s\n" % (
+                self.GREEN,
+                self.formatTime(record),
+                self.RESET,
+                level_prefix,
+                record.levelname.title(),
+                self.RESET,
+                message_prefix,
+                self.mask_secrets(record.getMessage()),
+                self.RESET))
+        else:
+            self.stream.write("%s - <%s> -- %s\n" % (
+                self.formatTime(record),
+                record.levelname.title(),
+                self.mask_secrets(record.getMessage())))
 
     def mask_secrets(self, msg):
         for secret in secrets:
             msg = msg.replace(secret, "<%s>" % secrets[secret])
         return msg
+
+class LessThanFilter(logging.Filter):
+    def __init__(self, exclusive_maximum, name=""):
+        super(LessThanFilter, self).__init__(name)
+        self.max_level = exclusive_maximum
+
+    def filter(self, record):
+        return 1 if record.levelno < self.max_level else 0
+
+def SuriStreamHandler():
```

Comment:
I think a better name for this function would be `ConfigureLogging` or something along those lines.